spec/expression.dd

```diff
@@ -1675,7 +1675,6 @@ $(H3 $(LNAME2 super, super))
         cast to $(D this)'s base class.
         It is an error if there is no base class.
         It is an error to use the $(D super) reference outside a non-static class member function.
-        (Only class $(D Object) has no base class.)
```

Comment:
Seems oddly out of place. Is it even true? What about Objective-C classes?

Reply:
Yes, I think it is valuable information that `Object` does not have a base class. Instead of deleting this, it is better to add to the list. The list does not have to be complete, we can incrementally populate it.

Reply:
Objective-C is bound through interfaces. Once you create a class to implement that interface, it is a D class.

Reply:
What would be a best-effort list so far be?
“(The only `extern(D)` class without a base class is `Object`. Note that `extern(C++)` classes have no base class unless specified.“ (If I understand @adamdruppe correctly, there are no `extern(Objective-C)` classes anyway; I checked and the compile error said they were not supported.) Checking `extern(C)` classes, the spec says that it’s legal to write `extern(C) class`, but `extern(C)` is ignored then.